help/en/Acknowledgements.shtml

```diff
@@ -929,7 +929,7 @@
         to the Tables</li>
 
         <li>Steve Rawlinson, who fixed a bug in Layout Blocks and
-        generously provides the <a href="http://jmri.tagadab.com/jenkins/job/Development/job/Packages/">Jenkins server</a>
+        generously provides the <a href="http://builds.jmri.org/jenkins/job/Development/job/Packages/">Jenkins server</a>
```

Comment:
This is no longer correct.

It should be worded similar to:

'generously provided the previous Jenkins server'

i.e. no link as it is no longer from the Tagadab infrastructure.

Reply:
Thanks for the clarification. Will update the text shortly.